potato/flask_server.py

```diff
@@ -3827,6 +2544,7 @@ def run_create_task_cli():
     """
     Run create_task_cli().
     """
+    import webbrowser

```

Comment:
what's with the webbrowser import here?

Reply:
I refactored `main()` to either call `run_create_task_cli()` or `run_server()` -- my understanding is that `webbrowser` is only being used for `create_task_cli()`. I can move it back to importing at a module level.

Reply:
Oh right! I think we could actually remove that import and the code that uses it and replace it with a `raise Exception("Gui-based design not implemented yet")` kind of message. I think this pull request is all good to go then.